docs/changelog.md

```diff
@@ -18,6 +18,14 @@ permalink: /changelog/
     * `browser-engine-gecko-beta`: GeckoView 72.0
     * `browser-engine-gecko-nightly`: GeckoView 73.0
 
+* **feature-app-links**
+  * ⚠️ **This is a breaking change**:
+  * Feature now contains two parts.  One part is the AppLinksFeature, the other part is RequestInterceptor.
+  ```Kotlin
```

Comment:
nit: lowercase 'k' otherwise markdown doesn't recognize the highlighter language.

Reply:
Thanks, didn't know that.  Thought the correct format is uppercase.  I'll fix.

Reply:
We have some cases of uppercase in the changelog, but highlighting doesn't work on those.

Reply:
Should I fix them as well?